components/Pages/BrokerAssistant/SearchServiceUser/index.js

```diff
@@ -0,0 +1,127 @@
+import { useState } from 'react';
+import { BROKER_PORTAL_ROUTE, CARE_PACKAGE_ROUTE, LOGOUT_ROUTE } from '../../../../routes/RouteConstants';
+import { Button, Container, Header, Input, Breadcrumbs } from '../../../HackneyDS'
+import DatePicker from '../../../HackneyDS/DatePicker';
+import ServiceUserDetails from '../../BrokerPortal/ServiceUserDetails';
+import AlternativePagination from '../../../AlternativePagination';
+import Loading from '../../../Loading';
+import FormGroup from '../../../HackneyDS/FormGroup';
+import SearchResult from '../../../SearchResult';
+
+const links = [
+  { text: 'Broker Portal', href: BROKER_PORTAL_ROUTE },
+  { text: 'Care Charges', href: CARE_PACKAGE_ROUTE },
+  { text: 'Log Out', href: LOGOUT_ROUTE },
+];
+
+const breadcrumbs = [
+  { text: 'Home', href: '/' },
+  { text: 'Broker Portal', href: BROKER_PORTAL_ROUTE },
+  { text: 'Search for a service user' },
+];
+
+const SearchServiceUser = ({
+  searchResults,
+  pageNumber,
+  setPageNumber,
+  totalCount,
+  totalPages,
+}) => {
+  const [initialFilters] = useState({
+    postcode: '',
+    firstName: '',
+    lastName: '',
+    hackneyId: '',
+    dateOfBirth: null,
+  });
+  const [filters, setFilters] = useState({ ...initialFilters });
```

Comment:
Actually no need to use spread here. `useState` do not mutate initial object. Same below
```suggestion
  const [filters, setFilters] = useState(initialFilters);
```